.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+name: Build
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'npm'
+      - name: Run install
+        run: npm ci
+      - name: Build
+        run: npm run build
+      - name: upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: dist
+  build-storybook:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'npm'
+      - name: Run install
+        run: npm ci
+      - name: Build Storybook
+        run: npm run build-storybook -o dist/storybook
+      - name: upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: dist
+  
+      - name: Save PR number
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          mkdir -p ./pr
+          echo $PR_NUMBER > ./pr/pr_number
+      - uses: actions/upload-artifact@v4
```

Comment:
Is there a reason you used a different version than the rest?
i think v3 is the latest stable version

Reply:
Hmm, I didn't notice the version - I just copy & pasted it from somewhere online. Thanks for noticing that.
Actually, seems like v3 is about to be deprecated
https://github.com/actions/upload-artifact?tab=readme-ov-file#actionsupload-artifact
![image](https://github.com/hasadna/open-bus-map-search/assets/11145132/822c9320-b77a-4455-9068-bb97a5bafc0e)

I guess we could make an issue to upgrade the upload thing everywhere


Thanks for reviewing this!

Reply:
I'll merge this PR, let's open an issue if any further improvements required